courses/models.py

```diff
@@ -15,11 +15,20 @@ class Instructor(models.Model):
         User,
         on_delete=models.CASCADE
     )
+    autobiography = models.TextField(default='')
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
```

Comment:
Should we restrict this field so that it must be one of the categories listed in the project description? Mergers and Acquisitions; Markets; Risk Management; Securities; Financial Modelling; Operations; Fintech; and Information Technology.

Reply:
I'm thinking about only allowing the admin (superuser) to add any category.

Instructors will only be allowed to choose the category for their course from a drop-down list consisting predefined categories.